nerdlets/util/async.js

```diff
@@ -232,19 +232,19 @@ async function processBatchOfCardinalities(
 ) {
   let failedBatch = false;
   try {
-    const { data, errors } = await calculateCardinalityForRuleBatch(
+    const { data, error } = await calculateCardinalityForRuleBatch(
```

Comment:
Was `calculateCardinalityForRuleBatch` also updated to return `error` instead of `errors`? I think this may be a local function and not part of the changes

Reply:
`calculateCardinalityForRuleBatch` was not updated, but the response from NerdGraphQuery is: 
https://github.com/newrelic/nr1-metrics-aggregator/blob/37f6903c633d191b0728986e95b7dd26d3f573ea/nerdlets/util/async.js#L92-L95

`calculateCardinalityForRuleBatch` returns the response from `NerdGraphQuery`, which has the `errors` depreciation.

Reply:
Sweet!!!